MSBLOC.Infrastructure/Interfaces/IAccessTokenRepository.cs

```diff
@@ -1,11 +1,16 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using System.Threading.Tasks;
 using MSBLOC.Infrastructure.Models;
 
 namespace MSBLOC.Infrastructure.Interfaces
 {
-    public interface IAccessTokenRepository : IRepository<AccessToken, Guid>
+    public interface IAccessTokenRepository
```

Comment:
I still think this should inherit  `IRepository<AccessToken, Guid>`.

As two of these members are already in that interface, and it may be nice to get the free functionality of the IRepository as well as the additional add on Entity specific members here.

Reply:
Cool. We can attack this again later.